src/main/java/hudson/plugins/msbuild/MsBuildBuilder.java

```diff
@@ -32,6 +32,7 @@
 import hudson.util.ArgumentListBuilder;
 import org.jenkinsci.Symbol;
 import org.kohsuke.stapler.DataBoundConstructor;
+import javax.annotation.Nonnull;
```

Comment:
I could not find the appropriate import. Looked up here: https://stackoverflow.com/a/42695253/9493145

Reply:
My mistake for using the wrong symbol in my comment.  The transformation is described in https://www.jenkins.io/doc/developer/tutorial-improve/replace-jsr-305-annotations/ but since that was not needed in this repository, you would not have seen that description.  Sorry I made that mistake!
```suggestion
import edu.umd.cs.findbugs.annotations.NonNull;
```